src/WormholeJoin.sol

```diff
@@ -198,7 +198,7 @@ contract WormholeJoin {
             vat.move(address(this), vow, fee * RAY);
         }
         if (operatorFee > 0) {
-            vat.move(address(this), bytes32ToAddress(wormholeGUID.operator), operatorFee * RAY);
+            daiJoin.exit(bytes32ToAddress(wormholeGUID.operator), operatorFee);
```

Comment:
the only reason for not exiting right away by the operator would be to save gas and do batch exit, right?

Reply:
Potentially that could save a bit, but seems like more trouble than it's worth.